src/inc/quic_platform_winuser.h

```diff
@@ -519,8 +519,11 @@ CxPlatRefDecrement(
 
 typedef HANDLE CXPLAT_EVENT;
 
-#define CxPlatEventInitialize(Event, ManualReset, InitialState) \
-    *(Event) = CreateEvent(NULL, ManualReset, InitialState, NULL)
+#define CxPlatEventInitialize(Event, ManualReset, InitialState)         \
+    do {                                                                \
+        *(Event) = CreateEvent(NULL, ManualReset, InitialState, NULL);  \
+        CXPLAT_FRE_ASSERT(*Event != NULL);                              \
```

Comment:
I don't want to make a functional change here. Do a DBG (or maybe TEL) assert for now please. Same for below.